packages/radio/stories/radio.stories.tsx

```diff
@@ -23,7 +23,17 @@ export const Readonly = () => (
     I'm a readonly radio
   </Radio>
 )
-
+export const StyledRadio = () => (
```

Comment:
Could we maybe have a similar story for `Checkbox`?

Reply:
added! thanks for the suggestion!